templates/page.html

```diff
@@ -122,26 +123,28 @@ <h1 class="article-title">
                 {%- endif -%}
 
                 {%- if author_list | length == 1 -%}
-                    {%- set author_string = author_list.0 -%}
+                    {%- set author_string = '<span class="p-author">' ~ author_list.0 ~ '</span>' -%}
                 {%- else -%}
                     {%- set last_author = author_list | last -%}
                     {%- set other_authors = author_list | slice(end=-1) -%}
                     {%- set author_separator = macros_translate::translate(key="author_separator", default=", ", language_strings=language_strings) -%}
```

Comment:
We want to wrap the  _author_separator_ between a closing `</span>` and an opening `<span class="p-author">` tag, so that 
`name1, name2, name3` becomes `name1</span>, <span class="p-author">name2</span>, <span class="p-author">name3`. The the initial `<span class="p-author">` and terminal `</span>` being added at line 135.

```suggestion
                    {%- set author_separator = macros_translate::translate(key="author_separator", default=", ", language_strings=language_strings) -%}
                   {%- set author_separator = '</span>' ~ author_separator ~ '<span class="p-author">' -%}
```